setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+
+setup(
+    name="ops",
+    version="0.0.1",
+    url="https://github.com/canonical/operator",
+    packages=['ops'],
```

Comment:
We should be good netizens and add the ownership and licencing info in here, even at this early stage.

Reply:
We're still figuring out licensing. We don't need to block this PR based on that.

The URL should also be enough ownership information for now.

Reply:
That makes sense. I didn't mean to imply that we should block the PR, but raise the point for discussion :)